app/models/role.rb

```diff
@@ -0,0 +1,3 @@
+class Role < ApplicationRecord
+  validates :name, inclusion: { in: ['designer', 'producer'], message: "%<value>s is not a valid role" }
```

Comment:
Acho que tá sobrando um "s" e faltando o ponto final  na mensagem da validação @eduardotps. A mensagem deveria ser "%<value> is not a valid role."